librocksdb-sys/rocksdb_lib_sources.txt

```diff
@@ -136,6 +141,12 @@ options/options.cc
 options/options_helper.cc
 options/options_parser.cc
 port/port_posix.cc
+port/win/env_default.cc
+port/win/env_win.cc
+port/win/io_win.cc
+port/win/port_win.cc
+port/win/win_logger.cc
+port/win/win_thread.cc
```

Comment:
Should remove the Windows-specific files here. See the collapsed comment https://github.com/rust-rocksdb/rust-rocksdb/pull/593#pullrequestreview-864458314 where I made the same mistake.

Reply:
Thanks! Removed.